de.dlr.sc.virsat.model.extension.requirements.ui/src/de/dlr/sc/virsat/model/extension/requirements/ui/celleditor/RequirementsAttributeValuePerColumnEditingSupport.java

```diff
@@ -0,0 +1,122 @@
+/*******************************************************************************
+ * Copyright (c) 2008-2019 German Aerospace Center (DLR), Simulation and Software Technology, Germany.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+/**
+ * 
+ */
+package de.dlr.sc.virsat.model.extension.requirements.ui.celleditor;
+
+import org.eclipse.emf.edit.domain.EditingDomain;
+import org.eclipse.jface.viewers.ColumnViewer;
+
+import de.dlr.sc.virsat.model.dvlm.categories.CategoryAssignment;
+import de.dlr.sc.virsat.model.dvlm.categories.propertyinstances.APropertyInstance;
+import de.dlr.sc.virsat.model.dvlm.categories.propertyinstances.ComposedPropertyInstance;
+import de.dlr.sc.virsat.model.dvlm.categories.util.CategoryAssignmentHelper;
+import de.dlr.sc.virsat.model.extension.requirements.model.AttributeValue;
+import de.dlr.sc.virsat.model.extension.requirements.model.Requirement;
+import de.dlr.sc.virsat.model.extension.requirements.model.RequirementAttribute;
+import de.dlr.sc.virsat.project.editingDomain.VirSatTransactionalEditingDomain;
+
+/**
+ * Editing support for table with columns for the different attributes
+ *
+ */
+public class RequirementsAttributeValuePerColumnEditingSupport extends AbstractAttributeValueEditingSupport {
+
+	protected final int attributeIndex;
+
+	protected final VirSatTransactionalEditingDomain domain;

```

Comment:
Maybe I am overlooking it or maybe it is happening in some downstream class but you never read from the variable domain.